src/vivarium_inputs/data_artifact/loaders.py

```
@@ -376,6 +373,24 @@ def get_risk_relative_risk(risk, location):
     return result
 
 
+def get_risk_population_attributable_fraction(risk, location):
+    if risk.distribution not in ['lognormal', 'normal', 'ensemble']:
+        result = None
```

This should raise an error, right? This function should only be called on continuous risks.  Move the check up into the main risk loader function.  Otherwise we'll end up storing keys with empty values in the artifact.

---

Still need to address this.

---

huh yeah, I don't know why I put the error on line 135 instead of here. will change that